From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Sat, 5 Sep 2020 15:01:48 +0100
Subject: spelling

---
 lib/srfi/166/write.scm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/srfi/166/write.scm b/lib/srfi/166/write.scm
index 7c58eae..2edb867 100644
--- a/lib/srfi/166/write.scm
+++ b/lib/srfi/166/write.scm
@@ -477,7 +477,7 @@
             (else
              (displayed (write-to-string obj))))))))))
 
-;; The default formatter for `written', overriden with the `writer'
+;; The default formatter for `written', overridden with the `writer'
 ;; variable.  Intended to be equivalent to `write', using datum labels
 ;; for shared notation iff there are cycles in the object.
 
